subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperHttpsIntegrationTest.groovy

```diff
@@ -16,17 +16,21 @@
 
 package org.gradle.integtests
 
+import org.gradle.api.JavaVersion
 import org.gradle.integtests.fixtures.TestResources
 import org.gradle.test.fixtures.keystore.TestKeyStore
 import org.gradle.test.fixtures.server.http.BlockingHttpsServer
+import org.gradle.test.fixtures.server.http.TestProxyServer
 import org.gradle.wrapper.Download
 import org.junit.Rule
+import spock.lang.Issue
 
 import static org.gradle.test.matchers.UserAgentMatcher.matchesNameAndVersion
 
 class WrapperHttpsIntegrationTest extends AbstractWrapperIntegrationSpec {
```

Comment:
❌ I've run your pr through our CI pipeline and got some test failures on Linux. Can you please check those out: https://builds.gradle.org/viewLog.html?buildId=34250652&tab=buildResultsDiv&buildTypeId=Gradle_Check_Quick_1_bucket50 (you should be able to log in as guest).

Reply:
New build containing the latest changes from master: https://builds.gradle.org/viewQueued.html?itemId=35015922

Reply:
I think if the test coverage is fixed then we can proceed with the merge.